integreat_cms/cms/templates/pages/page_sbs.html

```diff
@@ -72,8 +72,20 @@ <h1 class="heading">
                                disabled>
                         <div class="mt-4">
                             <label class="block mt-4 font-bold">{{ page_translation_form.content.label }}</label>
-                            <textarea id="source_translation_tinymce" cols="40" rows="10" class="tinymce_textarea" disabled>{{source_page_translation.content}}</textarea>
```

Comment:
The tinymce textarea had the advantage that all formatting etc. in html is already rendered and shown exactly like intended... (now, e.g. headings and links are styled differently) did it not work to show the additions/deletions in the tinymce field?
Also, I think the margins etc. should look similar on the left and right sides, to be able to better compare them.

Reply:
I implement it like this so that it is looks the same like it does in revision view, I'll try to make it work on an editor instance